include/functions.h

```diff
@@ -3183,7 +3183,7 @@ void func_80169590(GlobalContext* globalCtx, s16 param_2, s16 param_3);
 void func_80169600(GlobalContext* globalCtx, s16 param_2);
 // void func_80169668(void);
 Camera* Play_GetCamera(GlobalContext* globalCtx, s16 index);
-s32 func_8016970C(GlobalContext* globalCtx, s16 camId, Vec3f* at, Vec3f* eye);
+s32 Gameplay_CameraSetAtEye(GlobalContext* globalCtx, s16 camId, Vec3f* at, Vec3f* eye);
```

Comment:
```suggestion
s32 Play_CameraSetAtEye(GlobalContext* globalCtx, s16 camId, Vec3f* at, Vec3f* eye);
```
Just remembered that we have this conversation recently, but even though OoT uses `Gameplay_`, MM currently uses `Play_` which OoT might switch to anyway. So I think would be better for now.